.github/workflows/lint.yml

```diff
@@ -34,6 +34,7 @@ jobs:
         uses: super-linter/super-linter/slim@v6.7.0
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          FILTER_REGEX_EXCLUDE: ((^|/)doxygen-awesome|moose_doxy_header.html|doxygen.css|tabs.css)
```

Comment:
I take it these are files you've copied from elsewhere, hence why they don't meet the linting requirements? Might be wroth adding a note to this effect.

Reply:
Indeed they are - I've added a note